torch/csrc/jit/passes/onnx/scalar_type_analysis.cpp

```diff
@@ -300,6 +303,8 @@ static void RecoverScalarTypeForOutput(
   out->replaceAllUsesAfterNodeWith(cast_node, cast_node->output());
 }
 
+// This error found when exports transfo_xl model using add op in uint8 type:
+// https://github.com/huggingface/transformers/blob/b020a736c374460af1b34267283f957988350630/src/transformers/models/transfo_xl/modeling_transfo_xl.py#L936
```

Comment:
Would it make sense to describe the error in a bit more detail here?

Reply:
Yes, sure. Added more details.